### ISSUE TITLE
Selector should be able to get element on the page by accessibility criteria

### ISSUE THREAD

**Issue by helen-dikareva** [DevExpress/testcafe#7138]
### What is your Scenario?

I want to get an element by using 
- [ARIA role](https://www.w3.org/TR/wai-aria-1.2/#roles)
- [ARIA attributes](https://www.w3.org/TR/wai-aria-1.2/#aria-attributes)
- [accessible name](https://w3c.github.io/accname/#dfn-accessible-name).

### What are you suggesting?

Selector API should provide an opportunity to specify element based on this data

### What alternatives have you considered?

It could be available by a plugin for TestCafe 

### Additional context

-
